modules/nextflow/src/main/groovy/nextflow/scm/AssetManager.groovy

```diff
@@ -436,37 +438,46 @@ class AssetManager {
         return remoteHead?.getTarget()?.getName()?.substring(REMOTE_REFS_ROOT.length())
     }
 
-    @Memoized
     Manifest getManifest() {
-        getManifest0()
+        try {
+            def config = getConfig0()
+            def manifestOpts = config.manifest as Map ?: Collections.emptyMap()
+            return new Manifest(manifestOpts)
+        }
+        catch( Exception e ) {
+            log.warn "Cannot read project config -- Cause: ${e.message ?: e}"
+            return new Manifest(Collections.emptyMap())
+        }
```

Comment:
Likely this is not needed anymore, because the retry strategy works at a lower level

Reply:
what do you mean?